### DISCUSSION TITLE
Airflow Broken

### DISCUSSION THREAD

**Discussion by imranshaikmuma** [apache/airflow#17925] [Q&A]
### Apache Airflow version

2.0.2

### Operating System

Amazon Linux AMI

### Deployment

Managed Services (Astronomer, Composer, MWAA etc.)

### What happened

We changed the dag folder and reverted it back and updated the environment
It gave the following error
<img width="1444" alt="Screen Shot 2021-08-30 at 7 58 07 PM" src="https://user-images.githubusercontent.com/24683611/131420614-740fe59c-a53e-4b4f-9062-f496f01e87af.png">

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Definitely not enough information - you need to check logs and provide more details if you want any help.

**Comment by potiuk** [comment]
@imranshaikmuma: I understand you might have the impression (from our error message) that it is enough to open the issue, but you need to realise, that this is not nearly enough for us to be able to help you.

Here is the new error message that we are discussing to not mislead people. Maybe you will find it helpful. Let us know if it is, and if you can provide extra information or find a solution to your problem on your own:

-----------------------------------

Something bad has happened.

Airflow is used by many users, and it is very likely that others had similar problems and you can easily find
a solution to your problem.

Consider following those steps:

  * gather the relevant information (detailed logs with errors, reproduction steps, details of your deployment)

  * find similar issues using:
     * <b><a href="https://github.com/apache/airflow/discussions">GitHub Discussions</a></b>
     * <b><a href="https://github.com/apache/airflow/issues">GitHub Issues</a></b>
     * <b><a href="https://stackoverflow.com/questions/tagged/airflow">Stack Overflow</a></b>
     * the usual search engine you use on a daily basis

  * if you run Airflow on a Managed Service, consider opening an issue using the service support channels

  * if you tried and have difficulty with diagnosing and fixing the problem yourself, consider creating a <b><a href="https://github.com/apache/airflow/issues/new/choose">bug report</a></b>.
    Make sure however, to include all relevant details and results of your investigation so far.

 -----------------------------
